docs/spec/connection-handshake/interfaces.md

```diff
@@ -57,17 +53,25 @@ function connOpenTry(
 
 ### 2. Using abstractions
 
-We will use the following two abstractions to parametrize the `connOpenTry` handler:
+We will use the following three abstractions to parametrize the `connOpenTry` handler:
 
-- the `ConnectionEnd` interface (cf. ICS 003):
+- The `Party` type:
 
 ```typescript
-interface ConnectionEnd {
-  state: ConnectionState
-  counterpartyConnectionIdentifier: Identifier
-  counterpartyPrefix: CommitmentPrefix
+interface Party {
+  connectionIdentifier: Identifier
   clientIdentifier: Identifier
-  counterpartyClientIdentifier: Identifier
+  prefix: CommitmentPrefix
+}
+```
+
+- A `Connection` type:
+
+```typescript
+interface Connection {
+  state: ConnectionState
```

Comment:
This is the state of the remote/ counterparty end.

Reply:
Yep. Tough I'm leaning towards removing the state field altogether..